python/pyspark/pandas/plot/core.py

```diff
@@ -181,14 +182,31 @@ def compute_hist(psdf, bins):
         colnames = sdf.columns
         bucket_names = ["__{}_bucket".format(colname) for colname in colnames]
 
+        # TODO(SPARK-49202): register this function in scala side
```

Comment:
will register this ML function in SparkSessionExtensions after https://github.com/apache/spark/pull/47637/files get merged

Reply:
using a python udf for now